src/main/scala/io/sdkman/changelogs/MavenMigration.scala

```diff
@@ -6,16 +6,15 @@ import com.mongodb.client.MongoDatabase
 @ChangeLog(order = "013")
 class MavenMigration {
   @ChangeSet(
-    order = "036",
-    id = "036-maven-3.9.6",
+    order = "037",
+    id = "037-maven-4.0.0-alpha-8",
```

Comment:
Shouldn't that be `9`?

```suggestion
    id = "037-maven-4.0.0-alpha-9",
```